nodeup/pkg/distros/distribution.go

```diff
@@ -52,6 +53,8 @@ func (d Distribution) BuildTags() []string {
 		t = []string{"_rhel7"}
 	case DistributionCoreOS:
 		t = []string{"_coreos"}
+	case DistributionFlatcar:
+		t = []string{"_flatcar"}
```

Comment:
Just FYI we hopefully aren't using these tags any more.  But no problem with leaving this in for now and removing once we confirm we're not using tags :-)